examples/apis/methods.rb

```diff
@@ -27,7 +27,7 @@ class Application < Sinatra::Application
     end
   end
 
-  get '/v1/methods' do
+  get '/v2/methods' do
     methods = Mollie::Method.all(params[:offset], params[:count],
```

Comment:
I'm getting an exception when retrieving payment methods, possible because the way pagination works has changed.

```ruby
     Failure/Error:
       ::Mollie::Method.all(
           api_key: get_preference(:api_key),
           include: 'issuers'
       )

     Mollie::Exception:
       {"status":422,"title":"Unprocessable Entity","detail":"Non-existent query parameter \"offset\" for this API call. Did you mean: \"locale\"?","field":"offset","_links":{"documentation":{"href":"https://docs.mollie.com/guides/handling-errors","type":"text/html"}}}
```

Reply:
Thanks, I'll have a look at it. I'm going through the [migration guide](https://docs.mollie.com/migrating-v1-to-v2) and double check all changes.